### DISCUSSION TITLE
recommend iGATE /Digipeater for 1200 Baud

### DISCUSSION THREAD

**Discussion by VR2XKP** [SQ9MDD/TTGO-T-Beam-LoRa-APRS#85] [Q&A]
As the LoRa  APRS tracker support 1200 Baud, any recommend iGate /Digipeater solution for 1200 Baud ?

**Comment by silasmoeckel** [comment]
Yaac has been working well for me.  https://www.ka2ddo.org/ka2ddo/YAAC.html
http://xastir.org/ also worked when tested.  TCP KISS seems to work well so anything that supports that and thats most of them APRS software.
BT works fine with aprsdroid so should be fine with others.

**Comment by SQ9MDD** [comment]
APRX, best solution for DIGI & iGate you can build DIGI with multiple interfaces

**Comment by mattbk** [comment]
I have not tested it, but I think the digipeater software at https://github.com/lora-aprs/LoRa_APRS_iGate can be adjusted to fit 1200 baud. I made a few notes on the different LoRa setups at https://github.com/SQ9MDD/TTGO-T-Beam-LoRa-APRS/issues/29#issuecomment-926018368.
